systemd/crc-cluster-status.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -x
+
+export KUBECONFIG=/opt/kubeconfig
+
+function check_cluster_unhealthy() {
```

Comment:
name should be `check_cluster_healthy` and return value should be 0 in case successful and 1 in case failure.